src/stories/meetings/MeetingCard.stories.tsx

```diff
@@ -58,3 +58,10 @@ IntervieweeView.args = {
   ...baseProps,
   showInterviewee: false
 }
+
+export const WithDate: Story<Props> = Template.bind({})
+WithDate.args = {
+  ...baseProps,
+  startAt: new Date(2022, 12, 17, 3, 24, 0),
+  endAt: new Date(2022, 12, 17, 3, 24, 0)
```

Comment:
Seria bom ter uma hora diferente pro horário final, tipo uns 30 minutos depois do início

Reply:
Feito!